src/main/java/pl/edu/agh/ztis/newsanalyzer/DictReader.java

```diff
@@ -0,0 +1,47 @@
+package pl.edu.agh.ztis.newsanalyzer;
+
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DictReader {
+
+    private String filepath;
+    private String delimiter = ",";
+
+    public DictReader(String filepath) {
+        this.filepath = filepath;
+    }
+
+    public Map<String, String> read() {
+        Map<String, String> result = new HashMap<>();
+        BufferedReader br = null;
+        String line;
+
+        try {
+
+            br = new BufferedReader(new FileReader(this.filepath));
```

Comment:
Do we need this.filepath here? A filepath would be enough as it's a class field - more a cosmetic change :)